# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/firefox/hotfix-32bit.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

 1:17.50    Compiling wr_malloc_size_of v0.2.2 (/srv/t2/src.firefox.mipsel.250822.004221.9198/firefox-142.0/gfx/wr/wr_malloc_si
ze_of)
 1:17.55 error[E0412]: cannot find type `AtomicU64` in module `std::sync::atomic`
 1:17.55    --> gfx/wr/wr_malloc_size_of/lib.rs:508:24
 1:17.55     |
 1:17.55 508 |       std::sync::atomic::AtomicU64
 1:17.55     |                          ^^^^^^^^^ help: a struct with a similar name exists: `AtomicU16`
 1:17.55    --> /srv/t2/src.rustc.mipsel.250820.192550.27451/rustc-1.89.0-src/library/core/src/sync/atomic.rs:3710:1
 1:17.55    ::: /srv/t2/src.rustc.mipsel.250820.192550.27451/rustc-1.89.0-src/library/core/src/sync/atomic.rs:3727:1
 1:17.55     |
 1:17.55     = note: similarly named struct `AtomicU16` defined here

--- firefox-142.0/gfx/wr/wr_malloc_size_of/lib.rs.vanilla	2025-08-22 17:41:15.948438423 +0200
+++ firefox-142.0/gfx/wr/wr_malloc_size_of/lib.rs	2025-08-22 17:42:28.565598005 +0200
@@ -504,14 +504,14 @@
 malloc_size_of_is_0!(
     std::sync::atomic::AtomicU8,
     std::sync::atomic::AtomicU16,
-    std::sync::atomic::AtomicU32,
-    std::sync::atomic::AtomicU64
+    std::sync::atomic::AtomicU32
+    //, std::sync::atomic::AtomicU64
 );
 malloc_size_of_is_0!(
     std::sync::atomic::AtomicI8,
     std::sync::atomic::AtomicI16,
-    std::sync::atomic::AtomicI32,
-    std::sync::atomic::AtomicI64
+    std::sync::atomic::AtomicI32
+    //, std::sync::atomic::AtomicI64
 );
 
 malloc_size_of_is_0!(std::num::NonZeroUsize);
